curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-projects/demographic-data-analyzer.md

```diff
@@ -32,7 +32,7 @@ In this challenge you must analyze demographic data using Pandas. You are given
 
 You must use Pandas to answer the following questions:
 
-- How many people of each race are represented in this dataset? This should be a Pandas series with race names as the index labels. (`race` column)
+- How many people of each race are represented in this dataset? This should be a Pandas series with race names as the index labels and in descending order. (`race` column)
```

Comment:
I am not certain that this is true? Do you have a reproducer demonstrating this? I passed this part of the project without doing any sorting.

Reply:
I think my Replit when doing the exercise should be public by default. Look at line 14.
https://replit.com/@richard1979/boilerplate-demographic-data-analyzer#demographic_data_analyzer.py

Reply:
@jeremylt : I tried your solution. It is clearly more elegant. Using .value_counts() includes as sort in descending order, which if you solve this in a less elegant way is missed.
https://pandas.pydata.org/docs/reference/api/pandas.Series.value_counts.html

Reply:
I feel like this would be better addressed in the test suite, to be honest. The order should not really matter. That detail is not the point.